osrf/gazebo
# Backport #3031 - Make links within nested models modifiable from GUI Client

backport #3031 and a small follow-up PR #3059

To test:

1. source the gazebo setup file,
    ```
    source <install_path>/share/gazebo/setup.sh
    ```
1. launch the `multilink_shape_nested.world`, 
    ```
    GAZEBO_MODEL_PATH=$GAZEBO_MODEL_PATH:<path_to_gazebo>/test/models/ gazebo --verbose <path_to_gazebo>/test/worlds/multilink_shape_nested.world
    ```
1. Go to the left panel, expand `Models` > `multilink_nested::multilink`, and select `box_link`
1. Change its properties using the widget below, e.g. unchecking `gravity`, -> The properties should be modified successfully. Without the changes, an error msg should be printed in the console.
